qualtran/cirq_interop/_cirq_to_bloq.py

```diff
@@ -53,34 +54,36 @@ def _get_cirq_quregs(signature: Signature, qm: InteropQubitManager):
     return ret
 
 
-@frozen
-class CirqGateAsBloq(GateWithRegisters):
+class CirqGateAsBloqBase(GateWithRegisters):
```

Comment:
don't you need to mark this as abstract? even if the metaclass is inherited, it would probably help for documenting